api/opencl_platform_layer.asciidoc

```diff
@@ -1606,6 +1606,8 @@ include::{generated}/api/version-notes/CL_DEVICE_ATOMIC_MEMORY_CAPABILITIES.asci
         {CL_DEVICE_ATOMIC_ORDER_RELAXED} \| +
         {CL_DEVICE_ATOMIC_SCOPE_WORK_GROUP}
 
+        A device that only supports {CL_DEVICE_SVM_COARSE_GRAIN_BUFFER_anchor} as per {CL_DEVICE_SVM_CAPABILITIES_anchor} query can still report support for {CL_DEVICE_ATOMIC_SCOPE_ALL_DEVICES_anchor} in queries for {CL_DEVICE_ATOMIC_MEMORY_CAPABILITIES_anchor} and {CL_DEVICE_ATOMIC_FENCE_CAPABILITIES_anchor}. Operations parameterized with memory_scope_all_svm_devices on such a device will behave as if they were parameterized with memory_scope_device.
```

Comment:
I think it would be best to be consistent with [Section 3.3.7](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#memory-consistency-model) and describe this behavior in terms of [CL_MEM_SVM_ATOMICS](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#CL_MEM_SVM_ATOMICS) and/or [CL_DEVICE_SVM_ATOMICS](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#CL_DEVICE_SVM_ATOMICS).

How about something like:

> A device that does not support `CL_DEVICE_SVM_ATOMICS` (and hence does not support `CL_MEM_SVM_ATOMICS`) may still support `CL_DEVICE_ATOMIC_SCOPE_ALL_DEVICES`.  On these devices, an atomic operation with **memory_scope_all_svm_devices** will behave the same as if the scope were **memory_scope_device** - refer to (nice link to Section 3.3.7).

How do you feel about putting this text into a footnote?  The nice thing about a footnote is that we can include the same text both for both `CL_DEVICE_ATOMIC_MEMORY_CAPABILITIES` and `CL_DEVICE_ATOMIC_FENCE_CAPABILITIES`.

PS: The "anchor" attribute should only be used once, since it will define the target of the link for the enum.  Any other attributes - like those added by this PR - can just use the name of the attribute, instead.  I should really start writing that style guide...

Reply:
@lakshmih did you have a chance to look at this review comment?

Reply:
I suspect we should have similar text for `CL_DEVICE_ATOMIC_FENCE_CAPABILITIES` also, but this is a definite improvement, so let's cross that bridge if we come to it.